docs/user-guide/migration/pandas.md

```diff
@@ -147,8 +147,10 @@ called `hundredXValue` where the `value` column is multiplied by 100.
 In `Pandas` this would be:
 
 ```python
-df["tenXValue"] = df["value"] * 10
-df["hundredXValue"] = df["value"] * 100
+df.assign(
+    tenXValue=lambda df_: df_.value * 10,
+    hundredXValue=lambda df_: df_.value * 100
```

Comment:
I think the polars example should probably also use this syntax then, i.e.
```python
df.with_columns(
    tenXValue=pl.col("value") * 10,
    hundredXValue=pl.col("value") * 100,
)
```

Reply:
I'm not entirely certain, but it appears that using `.alias` is the recommended approach for renaming columns.

Reply:
both work https://pola-rs.github.io/polars/py-polars/html/reference/dataframe/api/polars.DataFrame.with_columns.html

this would just minimise the difference

Reply:
Considering this is a brief migration guide to help newcomers from `Pandas`, I agree with your perspective.